spec/system/remove_evidence_flow_spec.rb

```diff
@@ -6,10 +6,14 @@
     when_i_start_a_new_assessment
     and_i_choose_a_regular_confidence_level
     and_i_add_a_new_piece_of_evidence('Passport or travel document', 'UK passport')
+    and_i_answer_no_to_every_question
+    then_i_get_a_score('XX', 'out of 4')
     and_i_add_a_new_piece_of_evidence('Certificate', 'Marriage certificate')
-    then_i_remove_that_evidence('passport_uk')
-    then_i_cannot_see_that_evidence_in_the_overview('passport_uk') # TODO eventually 'UK Passport'
-    then_i_can_see_that_evidence_in_the_overview('certificate_marriage') # TODO eventually 'Marriage certificate'
+    and_i_answer_no_to_every_question
+    then_i_get_a_score('XX', 'out of 4')
```

Comment:
Maybe we can actually test that we are getting a score back?

Reply:
Yeah I didn't actually implement that yet. It would be fairly trivial to add, but I'd quite like to keep this PR shorter rather than longer..